src/core/cloudxr/python/__main__.py

```
@@ -45,13 +56,68 @@ def _parse_args() -> argparse.Namespace:
+    stop = asyncio.get_running_loop().create_future()
+
+    def on_signal() -> None:
+        if not stop.done():
+            stop.set_result(None)
+
+    loop = asyncio.get_running_loop()
+    for sig in (signal.SIGINT, signal.SIGTERM):
+        try:
+            loop.add_signal_handler(sig, on_signal)
+        except NotImplementedError:
+            pass
```

Ruff (0.15.6)

[warning] Use `contextlib.suppress(NotImplementedError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(NotImplementedError): ...`

(SIM105)